common/djangoapps/util/organizations_helpers.py

```diff
@@ -88,6 +88,16 @@ def get_course_organizations(course_id):
     return organizations_api.get_course_organizations(course_id)
 
 
+def get_course_organization_id(course_id):
+    """
+    Returns organization id for course or None if the course is not linked to an org
+    """
+    course_organization = get_course_organizations(course_id)
+    if course_organization:
+        return course_organization[0]['id']
```

Comment:
Can there be more than one org for a course? If so, how do we choose which id to return?

Reply:
I'm not sure if there can be more than one org for a course, its the same logic/behavior as before my changes, I just shifted this logic out from certificates/api.py. I'll try to check if any courses have more than one org.